core/templates/dev/head/css/oppia.css

```diff
@@ -682,6 +682,7 @@ textarea {
 .oppia-sidebar-menu-open .oppia-sidebar-menu {
   box-shadow: 1px 0 3px rgba(0,0,0,0.12), 1px 0 2px rgba(0,0,0,0.24);
   -webkit-transform: translate3d(0, 0, 0);
+  overflow-y: scroll;
```

Comment:
Good question. We generally make an exception for -webkit, -moz, etc. properties, since those really do conceptually belong with their unprefixed versions.